test-resources/bin/pdb-test-env

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
```

Comment:
It'd be nice to have a short comment at the top describing what this is for. Same with setup-pdb-pg.